ui/package.json

```diff
@@ -179,6 +180,7 @@
     "react-router": "^3.0.2",
     "react-router-redux": "^4.0.8",
     "react-scrollbars-custom": "^4.0.0-alpha.8",
+    "react-spring": "^8.0.27",
```

Comment:
added react-spring for design-specific styling requests made as part of the epic